src/docs/the-new-sharetribe/the-new-sharetribe-tutorial/tutorial-marketplace-branding/index.md

```diff
@@ -25,16 +25,16 @@ color.” This will allow you to select a color. You can use a color
 picker, click a color on the color gradient, or input an RGB coordinate
 or a hex value.
 
-In this case, we're going to use RGB coordinates. Choose 177,69,49 for
-R,G and B. This should give you the dark red brand color of Saunatime.
+In this case, we're going to use RGB coordinates. Choose 101,68,228 for
```

Comment:
Currently, when Biketribe gets created, the brand color RGB is 124,58,237. Is there a specific reason why we are not using the same brand color here?

Reply:
Synced IRL, let's use 124,58,237!